modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPaste.ts

```diff
@@ -0,0 +1,143 @@
+import { RealClipboard, RealMouse } from '@ephox/agar';
```

Comment:
This test won't be running on CI because it's missing the `Test` suffix.

- [ ] Rename the file to `CopyAndPasteTest.ts`